### PR TITLE
[player-3600]

### PR DESCRIPTION
**Pull request by aeng7**
-IMA iframe workaround will apply before showing ads now

### CODE REVIEW

**Comment by aeng7** [comment]
This isn't something we can unit test easily

File: js/google_ima.js [hunk 1]
```diff
@@ -1563,6 +1563,11 @@ require("../html5-common/js/utils/utils.js");
           };
 
         OO._.each(imaAdEvents, addIMAEventListener, this);
+
+        //Workaround of an issue on iOS where the IMA iframe is capturing clicks.
+        if(OO.isIos) {
```
Comment (jpfranco) [inline js/google_ima.js]: Missing space

Comment (aeng7) [reply js/google_ima.js]: Added

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**